::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 28 Jun 2016 21:35:44 +0200
Subject: [PATCH] s/u/build_base_machine: Arch indep libdir(s).

The same fixes should be applied on arm64, for example.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 systems/ubuntu/build_base_machine.sh | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/systems/ubuntu/build_base_machine.sh b/systems/ubuntu/build_base_machine.sh
index 1b42a79..b0f3740 100755
--- a/systems/ubuntu/build_base_machine.sh
+++ b/systems/ubuntu/build_base_machine.sh
@@ -54,11 +54,12 @@ apt-get -y install libnuma-dev
 # packages related to VM

 # a few manual fix up on a ubuntu
-cd /lib/x86_64-linux-gnu
+local arch=$(uname -m)
+cd /lib/${arch}-linux-gnu
 ln -sf libssl.so.1.0.0 libssl.so
 ln -sf libcrypto.so.1.0.0 libcrypto.so

-cd /usr/lib/x86_64-linux-gnu
+cd /usr/lib/${arch}-linux-gnu
 ln -sf libxml2.so.2 libxml2.so


